source/scss/_library/_objects.blocks.scss

```diff
@@ -74,6 +75,25 @@
   &__object {
     padding-top: $space-half;
   }
+
+  &__dek {
+    p {
+      line-height: 1.45;
+
+      @include media('<=medium') {
+        font-size: $font-size-s;
+      }
+    }
+  }
+
+  // Block with title on top
+  &--top-title {
+
+    .c-block__title {
+      margin-bottom: $space-half;
+      text-align: left;
```

Comment:
This is for the blocks that use the js to move the title to the top of the block and place it within the `block_media` portion. Adding `.u-spacing--half` to the `block_media` content would produce undesirable results for instances where there's more than one element within the `block_media` section that shouldn't have spacing, e.g. when tags are placed below images. 

This might get a little refactoring when I dig into using `order` for this layout.